src/Microsoft.ML.Transforms/Text/NgramHashingTransformer.cs

```diff
@@ -951,7 +952,7 @@ public sealed class ColumnOptions
                 }
                 FriendlyNames = null;
                 Name = name;
-                InputColumnNames = inputColumnNames;
+                InputColumnNamesArray = inputColumnNames;
```

Comment:
Not sure if we really need to rename this. If yes, should we do the same to all transforms/trainers public APIs?

Reply:
No. I wanted the *public* name to remain what it was, just over a different type. But I wasn't quite willing to change the *internal* implementation to work over that structure, since I've run into subtle issues with that approach before, and I was just trying to do a quick PR. (You'll note that this renaming is purely an internal change.) But I had to name what it had been *something* different, right?

But since I wanted the public surface to remain something named `InputColumnNames` (as it had been prior to my change), I needed the internal backing field to be named something else. `InputColumnNamesArray` seemed like an innocent enough name.

I changed no name anywhere in our public surface, just the apparent type from an external perspective. (As well as, inevitably, making this a property instead of a field.)